### DISCUSSION TITLE
Using llava with huggingface transformers

### DISCUSSION THREAD

**Discussion by johnwick123f** [haotian-liu/LLaVA#398] [Q&A] [answered]
I want to use a 7b llava model with huggingface but I can’t really find any docs to use it? Any help would be great!

**Comment by haotian-liu** [comment, marked as answer]
Hi, our code base are based on huggingface transformers. Can you elaborate a bit? Do you mean the inference endpoint? Or use transformers to load the model?

**Comment by johnwick123f** [comment]
Oh I just found out that this is based on transformers. Anyway, thanks for replying!
